### PR TITLE
210913 Nepal 2019 (2) JW

### PR DESCRIPTION
**Pull request by jianingwwww**
- add MR2 var to the scope considering the confusing definition of measles

see #9

### CODE REVIEW

**Review by robin-wang — APPROVED** [review]

File: 7_child_vaccination.do [hunk 1]
```diff
@@ -89,8 +89,17 @@
 			replace c_measles = . if ((inrange(im5,1,3) & (inrange(im6m1y,6667,9999) | inrange(im6m1d,97,98))) | (im11 == 1 & inlist(im26,8,9)) | (inlist(im2,8,9) & inlist(im11,8,9)))	// missing if measles DK/missing for card and memory
 			replace c_measles = . if ((inrange(im5,1,3) & (inrange(im6m2y,6667,9999) | inrange(im6m2d,97,98))) | (im11 == 1 & inlist(im26,8,9)) | (inlist(im2,8,9) & inlist(im11,8,9)))	// missing if measles DK/missing for card and memory
 		}
-	   
-	   
+		if inlist(country_name,"Serbia2019") {
```
Comment (robin-wang) [inline 7_child_vaccination.do:92]: Just to confirm that you only brought updates in Main back to Nepal branch? Or changes are also enforced onto Serbia and Algeria?

**Comment by robin-wang** [comment]
0913 Log - RW
Confirming Serbia / Algeria section in vaccination being only added for Main branch harmonisation, rather than legitimate updates from Jianing to the two respective survey branches.

File: 7_child_vaccination.do (hunk 1 above)
Comment (jianingwwww) [reply 7_child_vaccination.do:92]: > @jianingwwww
> Just to confirm that you only brought updates in Main back to Nepal branch? Or changes are also enforced onto Serbia and Algeria?

As far as I'm concerned and intended, this should only bring updates onto Nepal~

Comment (robin-wang) [reply 7_child_vaccination.do:92]: Brilliant!!!
Cheers!

**Pull request merged**